{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2018-1999018", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "Pydio version 8.2.1 and prior contains an Unvalidated user input leading to Remote Code Execution (RCE) vulnerability in plugins/action.antivirus/AntivirusScanner.php: Line 124, scanNow($nodeObject) that can result in An attacker gaining admin access and can then execute arbitrary commands on the underlying OS. This attack appear to be exploitable via The attacker edits the Antivirus Command in the antivirus plugin, and executes the payload by uploading any file within Pydio.", "edit_time": "Sat Oct 12 14:43:48 2019", "cve_code_des_url": "", "cve_other_info": "", "editor": "warcup", "cve_detail": "", "references": [{"URL": "https://www.mike-gualtieri.com/files/Pydio-8-VulnerabilityDisclosure-Jul18.txt", "Description": "MISC:https://www.mike-gualtieri.com/files/Pydio-8-VulnerabilityDisclosure-Jul18.txt"}], "cve_repaire": "", "cve_name_en": "cve_name_en", "add_time": "2018-07-23"}